.ci/install_nydus_snapshotter.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+target_dir="/opt/confidential-containers"
+
+nydus_snapshotter_repo=${nydus_snapshotter_repo:-"github.com/containerd/nydus-snapshotter"}
+nydus_snapshotter_repo_git="https://${nydus_snapshotter_repo}.git"
+nydus_snapshotter_version=${nydus_snapshotter_version:-"main"}
```

Comment:
v0.12.0 was released, we can and should use it here.
Better, we can get the version from kata-containers' versions.yaml file.

Reply:
Sure. I‘ll update it.

Reply:
Should I create a new PR in Kata for this?

Reply:
I have updated the version of snapshotter. As the `versions.yaml` file in kata-containers hasn't been updated yet, I'm temporarily providing the version here.